yt/yt/server/query_tracker/query_tracker_proxy.cpp

```diff
@@ -869,6 +871,25 @@ void TQueryTrackerProxy::AbortQuery(
         }
         THROW_ERROR error;
     }
+
+    // Save profile counter.
+    auto tags = TProfilingTags{
```

Comment:
it could be handy to extract construction of profiling tags from a record into a helper function.